src/styles/blocks/catalog.scss

```diff
@@ -11,7 +11,8 @@
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
-
+    perspective: 400px;
+    transition-duration: 300ms;
```

Comment:
set `transition-property` as well, it is better than default `all`
[transition-property](https://developer.mozilla.org/en-US/docs/Web/CSS/transition-property)